src/main/java/fr/polytech/sim/log/NamedLogger.java

```diff
@@ -16,4 +17,11 @@ protected NamedLogger(String name) {
         this.name = name;
     }
 
+     public void log(String format, Object... args) {
+         String entry = String.format(format, args);
+         String message = String.format("%s\t%s\n", this.name, entry);
+         output(message);
+     }
+
+     public abstract void output(String msg);
```

Comment:
Cette méthode dervait être `protected`, pour interdire l'accès depuis une classe quelconque.
```suggestion
     protected abstract void output(String msg);
```